frontend/talentsearch/src/js/components/applications/ApplicationCard.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { getPoolCandidateStatus } from "@common/constants/localizedConstants";
+import { relativeExpiryDate } from "@common/helpers/dateUtils";
+
+import { notEmpty } from "@common/helpers/util";
+import { type PoolCandidate, PoolCandidateStatus } from "../../api/generated";
+import getFullPoolAdvertisementTitle from "../pool/getFullPoolAdvertisementTitle";
+
+import ApplicationActions from "./ApplicationActions";
+import { canBeArchived, canBeDeleted, isDraft } from "./utils";
+import { type BorderMapKey, borderKeyMap, borderMap } from "./maps";
+
+export type Application = Omit<PoolCandidate, "pool" | "user">;
+
+export interface ApplicationCardProps {
+  application: Application;
+}
+
+const ApplicationCard = ({ application }: ApplicationCardProps) => {
+  const intl = useIntl();
+
+  const applicationIsDraft = isDraft(application.status);
+  const applicationCanBeArchived = canBeArchived(application.status);
+  const applicationCanBeDeleted = canBeDeleted(application.status);
+
+  let borderKey: BorderMapKey = "dt-gray";
+  if (!application.archivedAt && application.status) {
+    borderKey = borderKeyMap[application.status];
+  }
+
+  const border = borderMap[borderKey];
+
+  return (
+    <div data-h2-shadow="base(s)" {...border}>
+      <div
+        data-h2-padding="base(x1)"
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column) l-tablet(row)"
+        data-h2-justify-content="base(space-between)"
+      >
+        <div>
+          <h2 data-h2-font-size="base(h5)" data-h2-margin="base(0, 0, x0.5, 0)">
+            {application.poolAdvertisement
+              ? getFullPoolAdvertisementTitle(
+                  intl,
+                  application.poolAdvertisement,
+                )
+              : ""}
+          </h2>
+          <div data-h2-display="base(flex)" data-h2-gap="base(x0.5, all)">
+            <ApplicationActions.ContinueAction
+              show={applicationIsDraft}
+              application={application}
+            />
```

Comment:
isDraft is true for DRAFT and DRAFT_EXPIRED, I think it should only be for DRAFT as there will be no continuing on the expired one, or at least no point to it

Reply:
The way by which we determine this does only include `DRAFT`. I am considering changing it to a more simple approach like `status === PoolCandidateStatus.Draft`

https://github.com/GCTC-NTGC/gc-digital-talent/blob/bc58477649af790866caf3610716f0bc35073ea5/frontend/talentsearch/src/js/components/applications/utils.ts#L3-L5

Reply:
no, it seems to show it for expired too

![image](https://user-images.githubusercontent.com/40485260/188186580-e070ddfd-46b3-4692-82f5-73024ace28f2.png)

Reply:
Can you check the status of that application? It may have an expiry date that has passed but still have the `DRAFT` status.

Reply:
ah, sorry my bad
I was playing around with adminer and updated the pool-expiry but not the candidate-expiry, which was creating an impossible case (or atleast should be impossible)